moviesTime/Network/DetailsMovie.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  DetailsMovie.swift
+//  moviesTime
+//
+//  Created by Jose Eduardo on 31/07/21.
+//
+
+/*
+    "adult":false,
```

Comment:
when you are writing stuff in comments just to remember how the code will look like you need to remove them before uploading the code to the repo and if is not necessary anymore